nullaway/src/main/java/com/uber/nullaway/AbstractConfig.java

```diff
@@ -70,6 +70,8 @@
 
   protected boolean checkOptionalEmptiness;
 
+  protected String optionalClassPath;
```

Comment:
Should probably be an `ImmutableSet<String>`, like with `unannotatedClasses`. It should also be `@Nullable` (for the case where `checkOptionalEmptiness` is `false`). We want to support the case of: a) No Optional classes passed, b) One Optional class passed (or taken from the default), c) N Optional classes passed. This field would only support (b)

Reply:
Done 👍